ledger/error/generator/app/src/main/scala/com/daml/error/generator/app/ErrorCodeInventoryDocsGenApp.scala

```diff
@@ -58,6 +65,10 @@ object ErrorCodeInventoryDocsGenApp {
 
   }
 
+  private def newlineIntoSpace(s: String): String = {
+    s.replace('\n', ' ')
```

Comment:
Is there a chance for this to not work on Windows (maybe on some test when running on CI)? If yes, consider using [this](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/System.html#lineSeparator()).

Reply:
Only moved this code around (which as been working fine) so I'd like to avoid investing more time in it :(